### PR TITLE
KAA-1179 changes by review comments #4

### PR DESCRIPTION
**Pull request by nocs00**

### CODE REVIEW

**Comment by Polinaa** [comment]
Cassandra 2.2.5 in [744 line](https://github.com/kaaproject/kaa/pull/817/files#diff-305c6f2e4f42c9cceb8e2f2ff178483fR744).

File: doc/Administration-guide/System-installation/Single-node-installation/index.md [hunk 1]
```diff
@@ -44,11 +44,11 @@ Kaa requires the following third party components to be installed and configured
 * [Oracle JDK 8](http://www.oracle.com/technetwork/java/javase/downloads/index.html). Kaa has been tested on JDK 8.
 * [PostgreSQL 9.4](http://www.postgresql.org/download/)
 * [MariaDB 5.5](https://mariadb.com/)
-* [Zookeeper 3.4.5](http://zookeeper.apache.org/doc/r3.4.5/). Kaa requires ZooKeeper for coordination of server components.
+* [Zookeeper 3.4.5 or later](http://zookeeper.apache.org/doc/r3.4.5/). Kaa requires ZooKeeper for coordination of server components.
 
 Kaa has been tested on the latest production release of MariaDB and PostgreSQL.
 
-Kaa also requires [MongoDB 2.6.9](http://www.mongodb.org/downloads) or [Cassandra 3.5](http://cassandra.apache.org/download/) as a NoSQL database. The installation steps for third-party components are provided in the following section.
+Kaa also requires [MongoDB 2.6.9](http://www.mongodb.org/downloads) or [Cassandra 3.5](http://cassandra.apache.org/download/) (note: for RPM based OS (Centos) 3.0.7 is the latest available version in DataStax repo) as a NoSQL database. The installation steps for third-party components are provided in the following section.
```
Comment (Polinaa) [inline doc/Administration-guide/System-installation/Single-node-installation/index.md]: In [this installation guide](http://docs.kaaproject.org/display/KAA/Installation+guide) Cassandra 3.5 is used for CentOS.

**Comment by Polinaa** [comment]
I have some wrong numbering here: 
screenshot from 2016-06-30 13-33-21

**Comment by Polinaa** [comment]
You can omit brackets here:
screenshot from 2016-06-30 14-00-51

The next formatting may be applied:
screenshot from 2016-06-30 14-01-56

**Comment by nocs00** [comment]
Hi Polina,

please check changes, it covers all your comments. Thanks.

**Comment by Polinaa** [comment]
LGTM

File: doc/Administration-guide/System-installation/Single-node-installation/index.md [hunk 2]
```diff
@@ -450,11 +449,11 @@ cqlsh>
...
    There are 3 programs which provide 'java'.
-     Selection   Command
+   Selection   Command
    -----------------------------------------------
-      1         /usr/lib/jvm/jre-1.7.0-openjdk.x86_64/bin/java
-      2         /usr/lib/jvm/jre-1.6.0-openjdk.x86_64/bin/java
-    + 3         /usr/java/jdk1.8.0_60/bin/java
+    1      /usr/lib/jvm/jre-1.7.0-openjdk.x86_64/bin/java
+    2      /usr/lib/jvm/jre-1.6.0-openjdk.x86_64/bin/java
+   + 3      /usr/java/jdk1.8.0_60/bin/java
```
Comment (Polinaa) [inline doc/Administration-guide/System-installation/Single-node-installation/index.md:456]: remove `+`

File: doc/Administration-guide/System-installation/Single-node-installation/index.md [hunk 3]
```diff
@@ -469,184 +468,192 @@ cqlsh>
...
+   Stopping postgresql-9.4 service:            [  OK  ]
+   Starting postgresql-9.4 service:            [  OK  ]
+   ```
+
+   </div>
+   </div>
+   </li>
+   </ul>
 
 4. Install [Zookeeper 3.4.7](http://zookeeper.apache.org/doc/r3.4.5/).
```
Comment (Polinaa) [inline doc/Administration-guide/System-installation/Single-node-installation/index.md:658]: link is to 3.4.5

**Comment by Polinaa** [comment]
Some mismatches with active tabs for Ubuntu:
1) Mongo
screenshot from 2016-07-05 13-52-33

2) Cassandra
screenshot from 2016-07-05 13-52-54

**Pull request merged**